cockroachcloud/cloud-sso.md

```diff
@@ -5,16 +5,62 @@ toc: true
 docs_area: manage
 ---
 
-Users may authenticate to the [{{ site.data.products.db }} Console](https://cockroachlabs.cloud) using Single Sign-On (SSO) with GitHub, Google, or Microsoft as identity providers.
+Users may authenticate to the [{{ site.data.products.db }} Console](https://cockroachlabs.cloud) using Single Sign-On (SSO). GitHub, Google, and Microsoft are currently supported as identity providers (IdPs).
 
-Authentication with a centralized identity managed by a dedicated provider offers several security advantages:
+Additionally, [enhanced SSO features are currently available in Private Preview](#private-preview-enhanced-sso-features).
+
+Authentication with a centralized identity managed by a dedicated IdP offers several security advantages:
 
 - All supported SSO providers (Google, Microsoft, and GitHub) support multi-factor authentication (MFA).
 - Administrators avoid having to manage an additional set of credentials and tie those to other identities and credentials. Every additional credential or identity management operation introduces risk as well as costing effort, so minimizing these is doubly advantageous.
 - Administrators can onboard and offboard users quickly and efficiently.
 
-## Frequently Asked Questions (FAQ)
+## Private Preview: Enhanced SSO features
+
+### Support for SAML and OIDC identity protocols
+
+[Security Access Markup Language (SAML)](https://en.wikipedia.org/wiki/Security_Assertion_Markup_Language) and [OpenID Connect (OIDC)](https://openid.net/connect/) are identity protocols that allow applications to rely on IdPs for authentication.
+
+With support for SAML and OIDC, {{ site.data.products.db }} allows enterprise organizations to use a wide variety of self-hosted or SAAS enterprise IdP solutions, such as Okta, Active Directory, Onelogin.
+
+### Extended configuration options
+
+Enhanced SSO allows {{ site.data.products.db }} organization admins to configure a list of allowed and disallowed authentication methods.
+
+### Autoprovisioning
+
+This feature allows organization admins to share an organization-specific sign-up/sign-in URL. This removes the need to invite users to your organization.
+
+### How to enable Enhanced SSO for {{ site.data.products.dedicated }}
+
+File a [support ticket](https://support.cockroachlabs.com/) to enquire about the Enhanced SSO Private Preview.
+
+The following information will be required:
+
+- Customer name
+- CC Organization Id
+- Do you need auto-provisioning? [Yes / No]
```

Comment:
"auto-provisioning" or "Autoprovisioning" Have we set a precedent anywhere else in the docs on the hyphenation here?